README.md

```diff
@@ -27,6 +27,8 @@ If you're interested developing this application, follow these steps to get runn
 
 You will now be able to develop the frontend React application. Any changes made to the code will hot-reload upon save.
 
+Note: If the above is failing to compile on a Windows machine, you will first need to run `git fetch --all` and then `git reset --hard`. After successfully running these two commands, you can then retry step 4 (`npm start`).
```

Comment:
I thought you had to run the command to adjust how Windows handled line endings as well?